src/sxastarter/README.md

```diff
@@ -1,4 +1,6 @@
 # Sitecore JSS Next.js Sample Application
 
 
-[Documentation](https://doc.sitecore.com/xp/en/developers/hd/200/sitecore-headless-development/sitecore-javascript-rendering-sdk--jss--for-next-js.html).
+[Documentation (Experience Platform)](https://doc.sitecore.com/xp/en/developers/hd/210/sitecore-headless-development/sitecore-javascript-rendering-sdk--jss--for-next-js.html)
```

Comment:
should we use 211 here since docs have been published?

Reply:
I think so, done